editor/editor_log.cpp

```diff
@@ -214,6 +214,21 @@ void EditorLog::_meta_clicked(const String &p_meta) {
 			ScriptEditor::get_singleton()->edit(res, line - 1, 0);
 			InspectorDock::get_singleton()->edit_resource(res);
 		}
+	} else if (p_meta.contains(".cpp") || p_meta.contains(".h")) {
```

Comment:
Add support for macOS/iOS platform files:

```suggestion
	} else if (p_meta.contains(".cpp") || p_meta.contains(".h") || p_meta.contains(".mm")) {
```

You could also add `.hpp` but it's not used in any first-party Godot source file, only third-party files and godot-cpp.